format/index.md

```diff
@@ -134,15 +134,22 @@ empty.
 A document's top level **MAY** also have the following members:
 
 * `jsonapi`: an object describing the server's implementation
-* `links`: URLs related to the primary data.
+* `links`: a [links object][links] related to the primary data.
 * `included`: an array of resource objects that are related to the primary
   data and/or each other ("included resources").
 
-If any of these members appears in the top-level of a response, their values
+If any of these members appears in the top level of a response, their values
 **MUST** comply with this specification.
 
 If a document does not contain a top-level `data` key, then the `included` key **MUST NOT** be present either.
 
+The top-level [links object][links] **MAY** contain the following members:
+
+* `self`: the [link][links] that generated the current response document.
```

Comment:
"A link to this resource" might be a better description of `self`. Since a "link" can be an object, not just a URL string, it's not really accurate to say the link generated the current response document.